src/public/components/pretty-stringify.js

```diff
@@ -0,0 +1,7 @@
+import K, * as U from "karet.util"
+import React     from "karet"
+
+export const PrettyStringify = ( { space, obj } ) =>
+  <pre className="pretty-stringify">
+    { U.lift( x => JSON.stringify( x, null, Number( space ) ) )( obj ) }
+  </pre>
```

Comment:
I was actually previously going to suggest turning this into a proper component taking props. 👍

Note that written this way the `space` needs to be a (numeric) constant, which is something to watch out.  Also, when there is just a single component exported from a module I tend to export it as the default.

How about something like this:

```js
export const PrettyStringify = ( { value, replacer = null, space = null } ) =>
   <pre className="pretty-stringify">
     { U.lift(JSON.stringify)(value, replacer, space) }
   </pre>
```

This just kind of directly wraps `JSON.stringify` as a component.  All the parameters are supported with the normal defaults.  Perhaps `space` should be defaulted to `2` instead?  I took the prop names from the parameter names here: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

BTW, I find this idea of wrapping `JSON.stringify` neat.